kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/modem/huawei/HuaweiModemAtCommands.java

```diff
@@ -12,9 +12,9 @@
 
 public enum HuaweiModemAtCommands {
 
-    GET_ICCID("at^iccid?\r\n"),
+    GET_ICCID("at+iccid\r\n"),
     GET_SIM_TYPE("at^cardmode\r\n"),
-    DISABLE_URC("at^curc=0\r\n");
+    DISABLE_URC("at^dsci=0\r\n");
```

Comment:
Are the changes to this class that is for the Huawei modem related to the PR?

if there are further changes that need to be done in other section of the code, could you please open an issue and solve in another PR?

Thanks